src/main/java/org/fundacionjala/coding/cynthia/Video/Movie.java

```diff
@@ -0,0 +1,50 @@
+package org.fundacionjala.coding.cynthia.Video;
+
+/**
+ * movie class.
+ */
+abstract class Movie {
+    public static final int CHILDRENS = 2;
+    public static final int REGULAR = 0;
+    public static final int NEW_RELEASE = 1;
+    private String titleMovie;
+    private int priceCodeMovie;
```

Comment:
Now you need to remove the CHILDRENS, REGULAR, NEW_RELEASE constants and the priceCodeMovie attribute
and take advantage of inheritance. :)